From jmm@debian.org  Wed Jan 21 16:32:17 2009
From: Moritz Muehlenhoff <jmm@debian.org>
Date: Wed, 21 Jan 2009 22:00:42 +0100
Subject: Staging: wlan-ng: Remove stray comments in header files
To: greg@kroah.com
Cc: Moritz Muehlenhoff <jmm@debian.org>
Message-ID: <1232571646-32579-3-git-send-email-jmm@debian.org>


The header files contain a few comments, which describe the
function of the header element. Remove them where not needed.

Signed-off-by: Moritz Muehlenhoff <jmm@debian.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/p80211conv.h    |    3 ---
 drivers/staging/wlan-ng/p80211hdr.h     |    4 ----
 drivers/staging/wlan-ng/p80211ioctl.h   |    4 ----
 drivers/staging/wlan-ng/p80211meta.h    |   15 ---------------
 drivers/staging/wlan-ng/p80211metamib.h |   28 ----------------------------
 drivers/staging/wlan-ng/p80211metamsg.h |   28 ----------------------------
 drivers/staging/wlan-ng/p80211mgmt.h    |    8 --------
 drivers/staging/wlan-ng/p80211msg.h     |    6 ------
 drivers/staging/wlan-ng/p80211netdev.h  |    3 ---
 drivers/staging/wlan-ng/p80211req.h     |   12 ------------
 drivers/staging/wlan-ng/p80211types.h   |    4 ----
 drivers/staging/wlan-ng/prism2mgmt.h    |   15 ---------------
 12 files changed, 130 deletions(-)

--- a/drivers/staging/wlan-ng/p80211conv.h
+++ b/drivers/staging/wlan-ng/p80211conv.h
@@ -169,9 +169,6 @@ typedef struct wlan_snap
 struct wlandevice;
 
 /*================================================================*/
-/* Externs */
-
-/*================================================================*/
 /*Function Declarations */
 
 int skb_p80211_to_ether( struct wlandevice *wlandev, u32 ethconv,
--- a/drivers/staging/wlan-ng/p80211hdr.h
+++ b/drivers/staging/wlan-ng/p80211hdr.h
@@ -251,10 +251,6 @@ typedef union p80211_hdr
 
 
 /*================================================================*/
-/* Extern Declarations */
-
-
-/*================================================================*/
 /* Function Declarations */
 
 /* Frame and header lenght macros */
--- a/drivers/staging/wlan-ng/p80211ioctl.h
+++ b/drivers/staging/wlan-ng/p80211ioctl.h
@@ -91,10 +91,6 @@
 #define P80211_NL_MCAST_GRP_DIST	BIT2	/* Distribution system messages */
 
 /*================================================================*/
-/* Macros */
-
-
-/*================================================================*/
 /* Types */
 
 /*----------------------------------------------------------------*/
--- a/drivers/staging/wlan-ng/p80211meta.h
+++ b/drivers/staging/wlan-ng/p80211meta.h
@@ -58,9 +58,6 @@
 #define _P80211META_H
 
 /*================================================================*/
-/* System Includes */
-
-/*================================================================*/
 /* Project Includes */
 
 #ifndef _WLAN_COMPAT_H
@@ -68,12 +65,6 @@
 #endif
 
 /*================================================================*/
-/* Constants */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
 /* Macros */
 
 /*----------------------------------------------------------------*/
@@ -140,12 +131,6 @@ typedef struct catlistitem
 } catlistitem_t;
 
 /*================================================================*/
-/* Extern Declarations */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
 /* Function Declarations */
 
 /*----------------------------------------------------------------*/
--- a/drivers/staging/wlan-ng/p80211metamib.h
+++ b/drivers/staging/wlan-ng/p80211metamib.h
@@ -58,9 +58,6 @@
 #define _P80211METAMIB_H
 
 /*================================================================*/
-/* System Includes */
-
-/*================================================================*/
 /* Project Includes */
 
 #ifndef _WLAN_COMPAT_H
@@ -68,24 +65,6 @@
 #endif
 
 /*================================================================*/
-/* Constants */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
-/* Macros */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
-/* Types */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
 /* Extern Declarations */
 
 /*----------------------------------------------------------------*/
@@ -95,11 +74,4 @@
 extern catlistitem_t mib_catlist[];
 extern u32 mib_catlist_size;
 
-
-/*================================================================*/
-/* Function Declarations */
-
-/*----------------------------------------------------------------*/
-/* */
-
 #endif /* _P80211METAMIB_H */
--- a/drivers/staging/wlan-ng/p80211metamsg.h
+++ b/drivers/staging/wlan-ng/p80211metamsg.h
@@ -58,9 +58,6 @@
 #define _P80211METAMSG_H
 
 /*================================================================*/
-/* System Includes */
-
-/*================================================================*/
 /* Project Includes */
 
 #ifndef _WLAN_COMPAT_H
@@ -68,24 +65,6 @@
 #endif
 
 /*================================================================*/
-/* Constants */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
-/* Macros */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
-/* Types */
-
-/*----------------------------------------------------------------*/
-/* */
-
-/*================================================================*/
 /* Extern Declarations */
 
 /*----------------------------------------------------------------*/
@@ -95,11 +74,4 @@
 extern catlistitem_t msg_catlist[];
 extern u32 msg_catlist_size;
 
-
-/*================================================================*/
-/* Function Declarations */
-
-/*----------------------------------------------------------------*/
-/* */
-
 #endif /* _P80211METAMSG_H */
--- a/drivers/staging/wlan-ng/p80211mgmt.h
+++ b/drivers/staging/wlan-ng/p80211mgmt.h
@@ -101,9 +101,6 @@
 #define _P80211MGMT_H
 
 /*================================================================*/
-/* System Includes */
-
-/*================================================================*/
 /* Project Includes */
 
 #ifndef  _WLAN_COMPAT_H
@@ -542,11 +539,6 @@ typedef struct wlan_fr_deauthen
 
 } wlan_fr_deauthen_t;
 
-
-/*================================================================*/
-/* Extern Declarations */
-
-
 /*================================================================*/
 /* Function Declarations */
 
--- a/drivers/staging/wlan-ng/p80211msg.h
+++ b/drivers/staging/wlan-ng/p80211msg.h
@@ -49,9 +49,6 @@
 #define _P80211MSG_H
 
 /*================================================================*/
-/* System Includes */
-
-/*================================================================*/
 /* Project Includes */
 
 #ifndef _WLAN_COMPAT_H
@@ -65,9 +62,6 @@
 #define WLAN_DEVNAMELEN_MAX	16
 
 /*================================================================*/
-/* Macros */
-
-/*================================================================*/
 /* Types */
 
 /*--------------------------------------------------------------------*/
--- a/drivers/staging/wlan-ng/p80211netdev.h
+++ b/drivers/staging/wlan-ng/p80211netdev.h
@@ -105,9 +105,6 @@
 #define P80211_NSDCAP_NOSCAN                0x200  /* nsd can scan */
 
 /*================================================================*/
-/* Macros */
-
-/*================================================================*/
 /* Types */
 
 /* Received frame statistics */
--- a/drivers/staging/wlan-ng/p80211req.h
+++ b/drivers/staging/wlan-ng/p80211req.h
@@ -49,18 +49,6 @@
 #define _LINUX_P80211REQ_H
 
 /*================================================================*/
-/* Constants */
-
-/*================================================================*/
-/* Macros */
-
-/*================================================================*/
-/* Types */
-
-/*================================================================*/
-/* Externs */
-
-/*================================================================*/
 /* Function Declarations */
 
 int	p80211req_dorequest(wlandevice_t *wlandev, u8 *msgbuf);
--- a/drivers/staging/wlan-ng/p80211types.h
+++ b/drivers/staging/wlan-ng/p80211types.h
@@ -58,10 +58,6 @@
 #define _P80211TYPES_H
 
 /*================================================================*/
-/* System Includes */
-/*================================================================*/
-
-/*================================================================*/
 /* Project Includes */
 /*================================================================*/
 
--- a/drivers/staging/wlan-ng/prism2mgmt.h
+++ b/drivers/staging/wlan-ng/prism2mgmt.h
@@ -62,15 +62,6 @@
 
 
 /*=============================================================*/
-/*------ Constants --------------------------------------------*/
-
-/*=============================================================*/
-/*------ Macros -----------------------------------------------*/
-
-/*=============================================================*/
-/*------ Types and their related constants --------------------*/
-
-/*=============================================================*/
 /*------ Static variable externs ------------------------------*/
 
 extern int	prism2_debug;
@@ -146,10 +137,4 @@ void prism2sta_processing_defer(struct w
 void prism2sta_commsqual_defer(struct work_struct *data);
 void prism2sta_commsqual_timer(unsigned long data);
 
-/*=============================================================*/
-/*--- Inline Function Definitions (if supported) --------------*/
-/*=============================================================*/
-
-
-
 #endif
